orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartStageHandler.kt

```diff
@@ -203,7 +203,7 @@ class StartStageHandler(
       return false
     }
 
-    val clonedContext = objectMapper.convertValue(this.context, Map::class.java) as Map<String, Any>
+    val clonedContext: MutableMap<String, Any> = this.context.toMutableMap()
     val clonedStage = Stage(this.execution, this.type, clonedContext).also {
```

Comment:
Because `clonedContext` was really `this.context`, this had the effect of mutating the current stage context via the `Stage()` constructor call in a way that's critical for some functionality, notably stripping `requisiteStageRefids` and `refId`.